doc/rst/usingchapel/tasks.rst

```diff
@@ -300,6 +300,14 @@ CHPL_TASKS == fifo
   can result in deadlock if there are not enough threads to implement
   all of the required tasks.
 
+  When ``CHPL_RT_NUM_THREADS_PER_LOCALE`` is set, a warning is issued like:
```

Comment:
"When ``CHPL_RT_NUM_THREADS_PER_LOCALE`` is set" -> "When ``CHPL_RT_NUM_THREADS_PER_LOCALE`` is set with fifo tasking"?

Reply:
Nvm, I now see this is under a fifo section.